invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/InvitationsMembersModal.js

```diff
@@ -0,0 +1,277 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Modal,
+  Tab,
+  Input,
+  Form,
+  Checkbox,
+  Item,
+  TextArea,
+  List,
+} from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+import { Image } from "react-invenio-forms";
+import _isEmpty from "lodash/isEmpty";
+import { withState } from "react-searchkit";
+import _upperFirst from "lodash/get";
+import { CommunityInvitationsApi } from "../../../api";
+import { ErrorMessage } from "../../components/ErrorMessage";
+import { SelectedMembers } from "./SelectedMembers";
+import { MembersSearchBar } from "./MemberSearchBar";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import CKEditor from "@ckeditor/ckeditor5-react";
+
+export class InvitationsMembersModal extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      selectedMembers: [],
+      role: undefined,
+      message: "",
+      error: undefined,
+      isLoading: false,
+      suggestions: [],
+    };
+  }
+
+  serializePeople = (people) => {
```

Comment:
```suggestion
  serializeUsers = (users) => {
```
if you serialize it for the dropdown I would include something about it in the name, so that we know the purpose just by reading it